docs/read.md

```diff
@@ -1,15 +1,59 @@
 # Read
 
-- [Basic Reads](#advanced-reads)
-- [Partial Reads](#partial-reads)
-- [Value Reads](#value-reads)
+- [Basic Reads](#basic-reads)
+  - [Read](#read)
+  - [Partial Reads](#partial-reads)
+  - [Value Reads](#value-reads)
+- [Advanced Reads](#advanced-reads)
+   - [Alias Reads](#alias-reads)
+   - [Derived Data Reads](#derived-data-reads)
 - [Query Syntax](#query-syntax)
-- [useCache](#advanced-reads)
+  - [Collections](#collections)
+  - [Single Document](#single-document)
+  - [Collection Group](#collection-group)
+  - [where](#where)
+  - [Where Clauses](#where-clauses)
+  - [Order By](#orderby)
+  - [Limit](#limit)
+  - [startAt/startAfter](#startatstartafter)
+  - [endAt/endBefore](#endatendbefore)
+- [Firestore Indexes](#firestore-indexes)
+- [Advanced Strategies](#advanced-strategies)
+  - [Rendering lists](#rendering-lists)
+  - [Pagination](#pagination)
+  - [Sliding Windows](#sliding-windows)
+  - [useCache](#usecache)
 
 ---
 
 ## Basic Reads
 
+**Read/Write Subscriptions**
+
+In Read/Write subscriptions are turned on by default. Anytime useRead is
+called it automatically listens for any document changes both locally and
+global and will rerender when data changes. As of v1, there is not a single
```

Comment:
Yes, our library manages data loading and updates. But the reader are all using it inside their React component. 

So the doc should be written to who the reader is. In this case it's a UI engineer wanting to know how it will affect their rendering. 

For the deep dive docs their purpose is for a library maintainer. So those show go deeper into when the data updates, where the strict vs shallow vs deep equality checks are and how they will impact the callee which is a React component and if it will do a costly dirty check.